lib/controller/controller.go

```diff
@@ -1162,7 +1161,7 @@ func (ctrl *ProvisionController) deleteVolumeOperation(volume *v1.PersistentVolu
 // getProvisionedVolumeNameForClaim returns PV.Name for the provisioned volume.
 // The name must be unique.
 func (ctrl *ProvisionController) getProvisionedVolumeNameForClaim(claim *v1.PersistentVolumeClaim) string {
-	return "pvc-" + string(claim.UID)
+	return fmt.Sprintf("%s-%s", "pvc", strings.Replace(string(claim.UID), "-", "", -1)[0:volumeNameUUIDLength])
```

Comment:
we can't change this. Instead, need to expose a parameter

Reply:
OK, got it